assets/src/components/detoursTable.tsx

```diff
@@ -41,7 +41,11 @@ export const DetoursTable = ({
 }: DetoursTableProps) => (
   <Table
     hover={!!data}
-    className={joinClasses([...classNames, "c-detours-table"])}
+    className={joinClasses([
+      ...classNames,
+      "c-detours-table",
+      status === DetourStatus.Active ? "table-active-detour" : null,
+    ])}
```

Comment:
suggestion(non-blocking if not small): is it possible to use the `variant` prop here?

Reply:
[That does seem to have worked](https://github.com/mbta/skate/pull/2788/commits/99a9a1368f0d6950170d425218c5e2dfeee3c6c1)